site/cypress/e2e/Cypress-Gradeable/office_hours_queue.spec.js

```diff
@@ -0,0 +1,213 @@
+const queueName = 'Cypress Office Hour Queue 1';
+const queueName_random = 'Cypress Office Hour Queue Random';
+const queueName1 = 'Cypress Office Hour Queue 2';
+const queueCode = 'cypress_test';
+const queueCode1 = 'cypress_test_fail';
+const newQueueCode = 'cypress_update';
+const enableQueue = () => {
+    cy.visit(['sample']);
+    cy.get('#nav-sidebar-course-settings').click();
```

Comment:
You can visit the page directly instead of clicking the sidebar element, as they are already tested in a different test. Apply throughout.

Reply:
done